src/Ide/Cradle.hs

```diff
@@ -58,6 +58,28 @@ findLocalCradle fp = do
   logm $ "Module \"" ++ fp ++ "\" is loaded by Cradle: " ++ show crdl
   return crdl
 
+implicitCradle :: FilePath -> IO (Cradle CabalHelper)
+implicitCradle fp = do
```

Comment:
@fendor i dont fully like this code, sure there will be a better one alternative. In my tests it seems to work though